presto-main/src/main/java/com/facebook/presto/execution/scheduler/PhasedExecutionSchedule.java

```diff
@@ -269,7 +268,7 @@ public Visitor(Collection<PlanFragment> fragments, DirectedGraph<PlanFragmentId,
         @Override
         public Set<PlanFragmentId> visitExchange(ExchangeNode node, PlanFragmentId currentFragmentId)
         {
-            checkArgument(node.getScope() == LOCAL, "Only local exchanges are supported in the phased execution scheduler");
+            checkArgument(node.getScope().isLocal(), "Only local exchanges are supported in the phased execution scheduler");
```

Comment:
Do we want to change this? (Or simply use `!node.getScope().isRemote()` ? )

Reply:
Let's keep `isLocal` method. It is easier to read than `!isRemote()`.